Example/Emission/ARRootViewController.m

```diff
@@ -230,6 +232,14 @@ - (ARCellData *)jumpToUserDocs
   }];
 }
 
+- (ARCellData *)jumpToArtwork
+{
+  return [self tappableCellDataWithTitle:@"Artwork" selection:^{
+    id viewController = [[ARArtworkComponentViewController alloc] initWithArtworkID:@"enrico-baj-portrait-1-from-baj-chez-picasso"];
```

Comment:
Since we're going to start with building out the non-commercial artwork, can we update this to the institution example below?

Reply:
Done!